.github/workflows/publish-to-pypi.yml

```diff
@@ -33,7 +34,4 @@ jobs:
     - name: Build package
       run: python -m build
     - name: Publish package
-      uses: pypa/gh-action-pypi-publish@27b31702a0e7fc50959f5ad993c78deac1bdfc29
-      with:
-        user: __token__
-        password: ${{ secrets.PYPI_API_TOKEN }}
+      uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
You will almost certainly need to make some updates in PyPI to use trusted publishing as a result of this change. Don't worry though, it's easy.